install/kubernetes/helm/istio/README.md

```diff
@@ -47,7 +47,12 @@ $ kubectl apply -f install/kubernetes/helm/helm-service-account.yaml
 $ helm init --service-account tiller
 ```
 
-3. Install Istio’s [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
+3. Create the `istio-system` namespace:
+```
+$ kubectl create ns istio-system
```

Comment:
What about defining environment variable `$ NAMESPACE=istio-system` directly after create the namespace, and use this variable in later commands.

So we can easily illustrate that `istio-system` is an example namespace, users can use other namespaces.

Reply:
Thanks. I didn't notice the $NAMESPACE is being defined below.
Updated.